tests/idmaps.bats

```diff
@@ -332,4 +333,5 @@ load helpers
 	if ! fgrep -q 'GIDs: [0]' <<< "$output" ; then
 		fgrep -q 'GIDs: [0, '$(id -g)']' <<< "$output"
 	fi
+	run storage unmount $mount
```

Comment:
This should be done implicitly by the `storage wipe` in the `teardown` function.  Does not doing this create test failures?

Reply:
Ah, the bind mount isn't being cleared in the Store's Wipe() method.

Reply:
yes, I must drop this change.  I've added only to check if it could help, but I am working on the real fix right now, as it is a regression in the PR code

Reply:
ok nevermind my last comment, I've commented only on your first one :-)

Still it is not enough, as the issue is in the code (unmount needs to read the uid/gid mapping used for the container)